packages/shared/src/components/Attachments/AttachmentItem.tsx

```diff
@@ -0,0 +1,219 @@
+                                                        setSelectedAsset(undefined);
+                                                    },
+                                                    initialValue: true,
+                                                    decorator: (
+                                                        <div className="tw-mr-2">
+                                                            <IconImageStack20 />
+                                                        </div>
+                                                    ),
+                                                },
+                                            ],
+                                        },
+                                        {
+                                            id: 'menu-delete',
+                                            menuItems: [
+                                                {
+                                                    id: 'delete',
+                                                    size: MenuItemContentSize.XSmall,
+                                                    title: 'Delete',
+                                                    style: MenuItemStyle.Danger,
+                                                    onClick: () => {
+                                                        onAttachmentDelete();
+                                                        setSelectedAsset(undefined);
+                                                    },
+
+                                                    initialValue: true,
+                                                    decorator: (
+                                                        <div className="tw-mr-2">
+                                                            <IconTrashBin20 />
+                                                        </div>
+                                                    ),
+                                                },
+                                            ],
+                                        },
+                                    ]}
+                                />
+                            </Flyout>
+                        </div>
+                    </div>
+                )}
+            </div>
+        );
+    }
+);
+
+AttachmentItem.displayName = 'AttachmentItem';
+
+export const SortableAttachmentItem = (props: SortableAttachmentItemProps) => {
+    const { attributes, listeners, setNodeRef, transform, transition, isDragging } = useSortable({
+        id: props.item.id,
+    });
+
+    const transformStyle = {
+        transform: transform ? `translate(${transform.x}px, ${transform.y}px)` : '',
+        transition,
+        zIndex: isDragging ? 2 : 1,
+    };
+
+    const draggableProps = { ...attributes, ...listeners };
+
+    return (
+        <AttachmentItem
+            ref={setNodeRef}
+            isDragging={isDragging}
+            transformStyle={transformStyle}
```

Comment:
are `transformStyle` required as styles instead of tailwind classes in order to be able to overwrite the design tokens?

Reply:
yep, these are dynamically generated values from the library, as far as I know tailwind classes do not support dynamic classnames.